docs/docs/statistics/power.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Power Analysis
+description: Power Analysis
+sidebar_label: Power Analysis
+slug: power
+---
+
+# Power
+
+:::note
+Power analysis is currently in beta, and is available to only select customers. Currently GrowthBook offers only frequentist power analysis. Bayesian power analysis is coming soon.
+:::
+
+## What is power analysis?
+
+Power analysis helps you estimate required experimental duration. Power is the probability of observing a statistically significant result, given your feature has some effect on your metric.
+
+## When should I run power analysis?
+
+You should run power analysis before your experiment starts, to determine how long you should run your experiment. The longer the experiment runs, the more users are in the experiment (i.e., your sample size increases). Increasing your sample size lowers the uncertainty around your estimate, which makes it likelier you achieve a statistically significant result.
+
+## What is a minimum detectable effect, and how do I interpret it?
+
+Recall that your relative effect size (which we often refer to as simply the effect size), is the percentage improvement in your metric caused by your feature. For example, suppose that average revenue per customer under control is \$100, while under treatment you expect that it will be $102. This corresponds to a ($102-$100)/$100 = 2% effect size. Effect size is often referred to as lift.
+
+Given the sample variance of your metric and the sample size, the minimum detectable effect (MDE) is the smallest effect size for which your power will be at least 80%.
+
+For example, suppose your MDE is 2%. If you feel like your feature could drive a 2% improvement, then your experiment is high-powered.
+
+## How do I run a power analysis?
+
+1. Navigate to app-growthbook.io/power-calculator.
```

Comment:
We don't want to use links like this because for self-hosted folks this won't work (and the link for cloud would be app.growthbook.io anyways). So consider instead explaining how to get there from the app itself + how to enable it given our plans to have it disabled by default in alpha.

Reply:
updated, thx